scout/server/blueprints/phenotypes/views.py

```diff
@@ -28,3 +28,16 @@ def api_hpo_terms():
 
     data = controllers.hpo_terms(store=store)
     return jsonify(data)
+
+
+@hpo_bp.route("/api/v1/phenotypes/search/<search_string>", methods=["GET"])
+@public_endpoint
+def api_hpo_term_search(search_string):
+    """Public API for HPO phenotype term: search for a HPO terms matching a string
+
+    Returns:
+        data(dict): result dict with key "phenotypes" set to an array of all matching phenotype terms
+    """
+
+    data = controllers.hpo_terms(store=store, query=search_string)
```

Comment:
The search doesn't work but the CORS error in the html disappears:

![image](https://user-images.githubusercontent.com/28093618/133372343-10302ffe-3162-4e08-842a-24a156fab64f.png)